docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

```diff
@@ -161,6 +161,10 @@ users: [
 
 {{pluck users "image.url"}}'
 <!-- => barney.jpg,fred.jpg -->
+
+Standard pluck helper example:

```

Comment:
Instead of the standard pluck helper example, maybe we show how to use it with the {{each}} helper?

```handlebars
{{#each (pluck users "user")}}hello {{this}} {{/each}}
<!-- => 'hello barney hello fred ' -->
```